example/rollup/rollup.config.js

```diff
@@ -0,0 +1,11 @@
+import resolve from "@rollup/plugin-node-resolve";
+
+export default {
+  input: "app.js",
+  output: {
+    sourcemap: false,
+    format: "es",
+    file: "app.output.js",
+  },
+  plugins: [resolve()],
```

Comment:
I tried this config and got an error, does here lost `{ browser: true }` option?